.github/workflows/test.yml

```diff
@@ -26,7 +26,7 @@ concurrency:
 # - django 4.2, python 3.11, mysql:8.1, parallel
 # - django 4.1, python 3.11, postgres:12, parallel, USE_EMAIL_USER_MODEL=yes
 # - django 4.2, python 3.12, postgres:15, parallel, DISABLE_TIMEZONE=yes
-# - django stable/5.0.x, python 3.10, postgres (allow failures)
```

Comment:
I wonder if we should keep testing against the `stable/5.0.x` branch as well (in addition to the released 5.0.x), so that we're quickly notified in the unlikely event of Django breaking something in an upcoming 5.0.x patch release. Maybe replace one of the 4.1 jobs so that we don't spin up yet another runner.

Reply:
added in ca25229